scripts/server_benchmark.py

```diff
@@ -11,17 +11,20 @@
 import argparse
 import cProfile
 
-import ga4gh.backend as backend
-import ga4gh.protocol as protocol
-import ga4gh.datarepo as datarepo
-
 import guppy
 
+import utils
+utils.ga4ghImportGlue()
+import ga4gh.backend as backend  # noqa
+import ga4gh.protocol as protocol  # noqa
+import ga4gh.datarepo as datarepo  # noqa
+
 
 class HeapProfilerBackend(backend.Backend):
-    def __init__(self, dataDir):
-        dataRepository = datarepo.FileSystemDataRepository(dataDir)
-        super(HeapProfilerBackend, self).__init__(dataRepository)
+    def __init__(self, registryDb):
+        repo = datarepo.SqlDataRepository(registryDb)
+        repo.open(datarepo.MODE_READ)
+        super(HeapProfilerBackend, self).__init__(repo)
```

Comment:
This is part of the first commit, which brings server_benchmark up to date. The PR makes more sense if you look at the commits separately. I didn't see much point in making two PRs, as this would slow things down. I should have made it more clear that there was two separate things happening here though.